### PR TITLE
[Openshift-Dedicated] Added acceptance related changes for openshift dedicated platform

### PR DESCRIPTION
**Pull request by kesavanvt**
Added changes related to openshift dedicated after the new additions of tests to acceptance tier https://github.com/red-hat-storage/ocs-ci/pull/3341

Signed-off-by: kesavan <kvellalo@redhat.com>

### CODE REVIEW

**Comment by kesavanvt** [comment]
@ebenahar

File: tests/manage/pv_services/test_raw_block_pv.py [hunk 1]
```diff
@@ -55,7 +56,13 @@ def raw_block_pv(self):
         """
         worker_nodes = node.get_worker_nodes()
         pvcs = list()
-        for size in ["500Mi", "10Gi", "1Ti"]:
+        size_mb = "500Mi"
+        size_gb = "10Gi"
+        if config.ENV_DATA["platform"].lower() == "openshiftdedicated":
+            size_tb = "0.05Ti"
```
Comment (vasukulkarni) [inline tests/manage/pv_services/test_raw_block_pv.py:62]: isn't this 50 Megabytes? Do you need to test it?

Comment (kesavanvt) [reply tests/manage/pv_services/test_raw_block_pv.py:62]: It is requesting for 50 GB, In Openshift dedicated we have a quota limitation of 100GB and any request beyond that will cause an issue. This is a temporary fix until the quota limitation in Openshift dedicated is removed. I am aware that `size_tb = "0.05Ti"` will anyway request size in GBs, completely removing `size_tb` requires more changes within the test.

Comment (ebenahar) [reply tests/manage/pv_services/test_raw_block_pv.py:62]: @kesavanvt see if you can use https://github.com/red-hat-storage/ocs-ci/blob/9c5a5474d62777e868b80894d6b0f3567a7b605d/ocs_ci/utility/utils.py#L2337

Comment (kesavanvt) [reply tests/manage/pv_services/test_raw_block_pv.py]: @ebenahar Thanks, made use of the above method for conversion.

**Review by ebenahar — APPROVED** [review]

**Review by Shrivaibavi — APPROVED** [review]
LGTM

**Pull request merged**